packages/banner/src/Banner.tsx

```diff
@@ -0,0 +1,242 @@
+import React from 'react';
+import QuestionMarkWithCircleIcon from '@leafygreen-ui/icon/dist/QuestionMarkWithCircle';
+import InfoWithCircleIcon from '@leafygreen-ui/icon/dist/InfoWithCircle';
+import WarningIcon from '@leafygreen-ui/icon/dist/Warning';
+import CheckmarkWithCircleIcon from '@leafygreen-ui/icon/dist/CheckmarkWithCircle';
+import XIcon from '@leafygreen-ui/icon/dist/X';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const Variant = {
+  Info: 'info',
+  Warning: 'warning',
+  Danger: 'danger',
+  Success: 'success',
+} as const;
+
+type Variant = typeof Variant[keyof typeof Variant];
+
+export { Variant };
+
+const baseBannerStyles = css`
+  position: relative;
+  display: flex;
+  min-height: 40px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 20px;
+  padding-right: 12px;
+  border-width: 1px 1px 1px 0px;
+  border-style: solid;
+  border-radius: 6px;
```

Comment:
So correct me if I'm wrong, but we intend to use this banner in full width contexts, I.e. at the top of the page. In that context it doesn't make sense to have rounded corners. so I'm curious how to reconcile this?

Reply:
We're trying to eliminate contexts that are true full-width (like the top-of-page banners in MMS). When we were speaking about full-width, I meant relative to it's parent in the DOM – essentially that it's `display: block;`.